src/components/Filters/Aggregation/AggregationBranch/AggregationBranch.js

```diff
@@ -0,0 +1,145 @@
+  const dispatch = useDispatch();
+  const [isOpen, setOpen] = useState(false);
+
+  // Find all query filters that refer to the field name
+  const allFilters = coalesce(query, fieldName, []);
+
+  // Do any of these values start with the key?
+  const keyFilters = allFilters.filter(
+    (aFilter) => aFilter.indexOf(item.key) === 0,
+  );
+
+  // Does the key contain the separator?
+  const activeChildren = keyFilters.filter(
+    (key) => key.indexOf(SLUG_SEPARATOR) !== -1,
+  );
+
+  const activeParent = keyFilters.filter((key) => key === item.key);
+
+  let checkedState = UNCHECKED;
+  if (activeParent.length === 0 && activeChildren.length > 0) {
+    checkedState = INDETERMINATE;
+  } else if (activeParent.length > 0) {
+    checkedState = CHECKED;
+  }
+
+  // Fix up the subitems to prepend the current item key
+  const buckets = subitems.map((sub) => ({
+    disabled: item.isDisabled,
+    key: slugify(item.key, sub.key),
+    value: sub.key,
+    // eslint-disable-next-line camelcase
+    doc_count: sub.doc_count,
+  }));
+
+  const liStyle = 'parent m-form-field m-form-field--checkbox body-copy';
+  const id = sanitizeHtmlId(`${fieldName} ${item.key}`);
+
+  const toggleParent = () => {
+    const subItemFilters = getAllFilters(item.key, subitems);
+
+    // Add the active filters (that might be hidden)
+    activeChildren.forEach((child) => subItemFilters.add(child));
+
+    if (checkedState === CHECKED) {
+      dispatch(removeMultipleFilters(fieldName, [...subItemFilters]));
+    } else {
+      // remove all of the child filters
+      const replacementFilters = allFilters.filter(
+        (filter) => filter.indexOf(item.key + SLUG_SEPARATOR) === -1,
+      );
+      // add self/ parent filter
+      replacementFilters.push(item.key);
+      dispatch(replaceFilters(fieldName, [...replacementFilters]));
+    }
+  };
+
+  if (buckets.length === 0) {
+    return <AggregationItem item={item} key={item.key} fieldName={fieldName} />;
+  }
+
+  return (
+    <>
+      <li
+        className={`aggregation-branch ${sanitizeHtmlId(item.key)} ${liStyle}`}
+      >
+        <input
+          type="checkbox"
+          aria-label={item.key}
+          disabled={item.isDisabled}
+          checked={checkedState === CHECKED}
+          className="flex-fixed a-checkbox"
+          id={id}
+          onChange={toggleParent}
+        />
+        <label
+          className={`toggle a-label ${checkedState === INDETERMINATE ? ' indeterminate' : ''}`}
+          htmlFor={id}
+        >
+          <span className="u-visually-hidden">{item.key}</span>
+        </label>
+        <button
+          className="flex-all a-btn a-btn--link"
+          onClick={() => setOpen(!isOpen)}
+        >
+          {item.key}
+          {isOpen ? getIcon('up') : getIcon('down')}
+        </button>
+        <span className="flex-fixed parent-count">
+          <FormattedNumber value={item.doc_count} />
+        </span>
+      </li>
+      {isOpen === false ? null : (
```

Comment:
can you reverse this 

{isOpen ? (<ul...) : null }

Reply:
Good call. This has been updated.